package.json

```diff
@@ -30,7 +30,7 @@
   ],
   "dependencies": {
     "@octokit/rest": "^18.0.6",
-    "babel-runtime": "^6.26.0",
+    "babel-runtime": "^7.11.2",
```

Comment:
"babel-runtime" is old and no "7.11.2" version exists. (It causes `npm ERR! notarget No matching version found for babel-runtime@^7.11.2.`) This should be `@babel/runtime` and its current version is 7.12.5, as follows:

```
    "@babel/runtime": "^7.12.5",
```